packages/app/src/cli/services/deploy.ts

```diff
@@ -327,3 +297,51 @@ async function outputCompletionMessage({
     customSections,
   })
 }
+
+async function outputUnifiedCompletionMessage(
+  deploymentMode: DeploymentMode,
+  uploadExtensionsBundleResult: UploadExtensionsBundleOutput,
+  app: AppInterface,
+) {
+  const linkAndMessage = [
+    {link: {label: uploadExtensionsBundleResult?.versionTag, url: uploadExtensionsBundleResult.location}},
+    uploadExtensionsBundleResult?.message ? `\n${uploadExtensionsBundleResult?.message}` : '',
+  ]
+  if (deploymentMode === 'unified') {
+    return uploadExtensionsBundleResult?.released
+      ? renderSuccess({
+          headline: 'New version released to users.',
+          body: linkAndMessage,
+          nextSteps: [
+            [
+              'Run',
+              {command: formatPackageManagerCommand(app.packageManager, 'versions list')},
+              'to see rollout progress.',
+            ],
+          ],
+        })
+      : renderInfo({
+          headline: 'New version created, but not released.',
+          body: [...linkAndMessage, `\n\nThis app version needs to pass Shopify review before it can be released.`],
+          nextSteps: [['Submnit this version for review fron the Partners Dashboard.']],
```

Comment:
```suggestion
          nextSteps: [['Submit this version for review from the Partners Dashboard.']],
```

Reply:
we are going to [remove next steps sections](https://github.com/Shopify/app-deploys/issues/517#issuecomment-1583171613) completely